ansible/roles/common/tasks/main.yml

```diff
@@ -64,6 +64,13 @@
     tags:
       - maintenance
 
+  - name: apt-get update
+    apt: update_cache=yes cache_valid_time=86400
+    become: true
+    tags:
+      - upgrade
+      - maintenance
+
```

Comment:
Seems like this should be _after_ `install aptitude`. Why'd you move it?

Reply:
oh i think i needed it before the nginx thing and moved it up too far?